client/src/components/JurisdictionAdmin/BatchRoundSteps/BatchRoundTallyEntry/BatchRoundTallyEntry.test.tsx

```diff
@@ -162,6 +164,7 @@ describe('Batch comparison data entry', () => {
       row2 = within(rows[2]).getAllByRole('cell')
       expect(row1[1]).toHaveTextContent('1')
       expect(row2[1]).toHaveTextContent('2')
+      screen.getByText(hasTextAcrossElements('Last edited by: ja@example.com'))
```

Comment:
I think this `hasTextAcrossElements` is no longer necessary

Reply:
Good catch!